qa/rpc-tests/lib/base58.py

```diff
@@ -0,0 +1,124 @@
+'''
```

Comment:
There's already a very similar version of this in contrib/testgen/base58.py.
The differences look like mostly format

Reply:
Yes. Actually, entire lib folder is taken from `sentinel`. We already discussed this issue with @OlegGirko. Presumably, the best way is to make this dash-specific code as python library with modules (like dash_hash, for example) in order to use it in the `sentinel` as well as in `rpc-tests`.
So far I'm preparing the wrapper for dash-specific functionality to add it to the `BitcoinTestFramework`.